checkov/common/images/graph/image_referencer_provider.py

```diff
@@ -30,11 +32,13 @@ def __init__(self, graph_connector: Union[igraph.Graph, networkx.DiGraph],
     def extract_images_from_resources(self) -> list[Image]:
         pass
 
-    def extract_nodes(self) -> networkx.Graph | igraph.Graph | None:
+    def extract_nodes(self) -> LibraryGraph | None:
         if self.graph_framework == 'IGRAPH':
             return self.extract_nodes_igraph()
-        else:  # the default value of the graph framework is 'NETWORKX'
+        elif self.graph_framework == 'NETWORKX':
             return self.extract_nodes_networkx()
+        else:
+            return self.extract_nodes_rustworkx()
```

Comment:
Do we want the default behavior to be rustworkx? Not sure if it matters, but with igraph the default was still networkx